### PR TITLE
Windows/Deployment: VAMT System Requirements

### LINKED ISSUE #5818: Operating System Requirements
Can you please update the Operating System Requirements to reflect Server 2008 R2 or later?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 987a771b-1edd-2c26-5330-8c422146543b
* Version Independent ID: 382d1672-f6b9-f53e-4aa3-748556b37088
* Content: [VAMT Requirements (Windows 10) - Windows Deployment](https://docs.microsoft.com/en-us/windows/deployment/volume-activation/vamt-requirements#feedback)
* Content Source: [windows/deployment/volume-activation/vamt-requirements.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/deployment/volume-activation/vamt-requirements.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @greg-lindsay
* Microsoft Alias: **greglin**

### PR DESCRIPTION
**Pull request by illfated**
**Description:**

As per user feedback in issue ticket #5818 (**Operating System Requirements**), it is desirable to allow for later versions of Windows Server to be shown in the list of Operating Systems under "Minimum system requirement".

There is also a visible defect in last row of the table, breaking the last one and a half sentence over to the first column of the next row.

Thanks to @laolive382 for posting this request and making me notice the (ever so slightly) broken table.

**Changes proposed:**
- change "2008 R2, or Windows Server 2012" to "2008 R2, Windows Server 2012, or later"
- remove line break in the last row to restore the row structure
- add MarkDown spacing in the table for readability/editing

**issue ticket closure or reference:**

Closes #5818

### CODE REVIEW

**Comment by illfated** [comment]
Rich diff, for visual view of the changes:
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/5845/files?short_path=6240ddf#diff-6240ddfda00a9b1812cfdf346bd29d95

**Comment by illfated** [comment]
@JohanFreelancer9 : If available, I would like a copy review. Feel free to invite others to confirm factual OS version support. See also https://docs.microsoft.com/en-us/windows/deployment/volume-activation/install-vamt -> ref. [Windows Server with Desktop Experience](https://docs.microsoft.com/nb-no/windows-server/get-started/getting-started-with-server-with-desktop-experience)) (Applies To: Windows Server 2016).

File: windows/deployment/volume-activation/vamt-requirements.md [hunk 1]
```diff
@@ -31,17 +31,16 @@ The Volume Activation Management Tool (VAMT) can be used to perform activations
...
-|Additional Requirements |<ul><li>Connection to a SQL Server database. For more info, see [Install VAMT](install-vamt.md).</li><li>PowerShell 3.0: For Windows 8, Windows 8.1, Windows 10, and Windows Server® 2012, PowerShell is included in the installation. For previous versions of Windows and 
-Windows Server, you must download PowerShell 3.0. To download PowerShell, go to [Download Windows PowerShell 3.0](https://go.microsoft.com/fwlink/p/?LinkId=218356).</li><li>If installing on Windows Server 2008 R2, you must also install .NET Framework 3.51.</li></ul> |
+| Item | Minimum system requirement |
+| ---- | ---------------------------|
+| Computer and Processor | 1 GHz x86 or x64 processor |
+| Memory | 1 GB RAM for x86 or 2 GB RAM for x64 |
+| Hard Disk | 16 GB available hard disk space for x86 or 20 GB for x64 |
+| External Drive | Removable media (Optional) |
+| Display | 1024x768 or higher resolution monitor |
+| Network |Connectivity to remote computers via Windows® Management Instrumentation (TCP/IP) and Microsoft® Activation Web Service on the Internet via HTTPS |
```
Comment (JohanFreelancer9) [inline windows/deployment/volume-activation/vamt-requirements.md]:
```suggestion
| Network |Connectivity to remote computers via Windows Management Instrumentation (TCP/IP) and Microsoft Activation Web Service on the Internet via HTTPS |
```

File: windows/deployment/volume-activation/vamt-requirements.md [hunk 2]
```diff
@@ -31,17 +31,16 @@ The Volume Activation Management Tool (VAMT) can be used to perform activations
...
+| Operating System | Windows 7, Windows 8, Windows 8.1, Windows 10, Windows Server 2008 R2, Windows Server 2012, or later. |
+| Additional Requirements | <ul><li>Connection to a SQL Server database. For more info, see [Install VAMT](install-vamt.md).</li><li>PowerShell 3.0: For Windows 8, Windows 8.1, Windows 10, and Windows Server® 2012, PowerShell is included in the installation. For previous versions of Windows and Windows Server, you must download PowerShell 3.0. To download PowerShell, go to [Download Windows PowerShell 3.0](https://go.microsoft.com/fwlink/p/?LinkId=218356).</li><li>If installing on Windows Server 2008 R2, you must also install .NET Framework 3.51.</li></ul> |
```
Comment (JohanFreelancer9) [inline windows/deployment/volume-activation/vamt-requirements.md]:
```suggestion
| Additional Requirements | <ul><li>Connection to a SQL Server database. For more info, see [Install VAMT](install-vamt.md).</li><li>PowerShell 3.0: For Windows 8, Windows 8.1, Windows 10, and Windows Server 2012, PowerShell is included in the installation. For previous versions of Windows and Windows Server, you must download PowerShell 3.0. To download PowerShell, go to [Download Windows PowerShell 3.0](https://go.microsoft.com/fwlink/p/?LinkId=218356).</li><li>If installing on Windows Server 2008 R2, you must also install .NET Framework 3.51.</li></ul> |
```

**Comment by illfated** [comment]
This is a bit weird. I don't get any option to commit the suggested changes, only a **[Resolve conversation]** button, so it looks like I will have to amend the PR from my own repository via Git Extensions and my local cloned fork.

**Comment by illfated** [comment]
> Rich diff, for visual view of the changes:
> https://github.com/MicrosoftDocs/windows-itpro-docs/pull/5845/files?short_path=6240ddf#diff-6240ddfda00a9b1812cfdf346bd29d95

Link is still valid. Copy review suggestions applied.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**